plugins/talk-plugin-rich-text/client/components/rte/RTE.css

```diff
@@ -15,6 +15,7 @@
   position: absolute;
   margin: 12px 0 0 12px;
   color: #bbb;
+  z-index: -1;
```

Comment:
Instead of moving the  `placeholder` behind, I think the better solution would be to disable _pointer events_ and _user selection_ using

```css
user-select: none;
pointer-events: none;
```

Would this solve your issue?

Reply:
Hey @cvle! Thanks for reviewing. I can confirm that this does actually do the trick. It's a much better solution. Let me make the changes.